os-config.tpl.yml

```diff
@@ -150,6 +150,7 @@ rancher:
       - /usr/bin/iptables:/sbin/iptables:ro
       - /media:/media:shared
       - /mnt:/mnt:shared
+      - /:/host
```

Comment:
its used to get access to the `/dev/vda1` partition that we create - udev works outside the container, but not inside